Sources/Tools/SoundDecompress/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+include_directories("../../Plasma/CoreLib")
+include_directories("../../Plasma/FeatureLib/inc")
+include_directories("../../Plasma/FeatureLib")
+include_directories("../../Plasma/NucleusLib/inc")
+include_directories("../../Plasma/NucleusLib")
+include_directories("../../Plasma/PubUtilLib/inc")
+include_directories("../../Plasma/PubUtilLib")
+
+set(SoundDecompress_SOURCES
+    plAllCreatables.cpp
+    plOldAudioFileReader.cpp
+    main.cpp
+)
+
+add_executable(SoundDecompress ${SoundDecompress_SOURCES})
+target_link_libraries(SoundDecompress CoreLib plResMgr plAudioCore)
+target_link_libraries(SoundDecompress ${STRING_THEORY_LIBRARIES})
```

Comment:
Looks like we also need to link DirectX (for DirectSound) on Windows when CWaveFile is enabled.